objectstorage/options.go

```diff
@@ -12,6 +13,10 @@ import (
 	"github.com/iotaledger/hive.go/kvstore/debug"
 )
 
+var (
+	overrideCacheTime = -1
```

Comment:
I don't like the solution and the global variable here at all.
Why don't you simply pass the correct cacheTime to every ObjectStorage instance the software that uses this library?
For example if you want to disable cacheTime in goshimmer unit tests, do it there, but don't mess around with this lib IMHO.

Reply:
I agree with you that the current solution is convoluted...

In my original solution this is actually what I started to do but @jonastheis told me to do it this way. I wanted to argue but decided not to..

But since you brought this up let's see what @jonastheis thinks

Reply:
Guys IMHO this way it's just really bad design. 

You would wonder why the cache is doesn't work if you pass correct values, but somewhere else in the code someone called this function...

Reply:
The initial idea was to create a simple way to override/disable all caches that are created anywhere in the project for testing purposes (unit and integration tests). I agree that it's not a pretty solution but doing it manually on a per object storage basis is cumbersome and prone to be forgotten when introducing a new OS somewhere in the codebase.

Reply:
Maybe we should have another quick brainstorming session and see how we can simplify it via GoShimmer codebase and leave the OS alone.

Reply:
I agree with @muXxer 

1. I think global variables are evil in general
2. you have a config that masks other configs which is yucky